formats/financial_releases_geoblocker/frontend/examples/financial_releases_not_permitted.json

```diff
@@ -1,6 +1,6 @@
 {
   "content_id": "054e8890-8cba-47fd-ab3b-32fc7a3ca2b3",
-  "base_path": "/financial-releases-geoblocker/not-permitted",
```

Comment:
@bradwright in our pull request (waiting to be merged) we have changed the live route to '/location/not-permitted' so dont think we need to change this one....